client/src/pages/Background.js

```diff
@@ -0,0 +1,306 @@
+    marginTop: "3rem",
+    paddingBottom: "1rem",
+  },
+  input: {
+    marginTop: "0.5rem",
+    marginBottom: "1rem",
+  },
+  nextStep: {
+    backgroundColor: "#0000ff",
+    color: "#ffffff",
+    borderRadius: 35,
+    marginTop: "3rem",
+    paddingLeft: "1.5rem",
+    paddingRight: "1.5rem",
+  },
+  question: {
+    marginTop: "2rem",
+    marginBottom: "0.5rem",
+  }
+});
+const labels = {
+  1: "JUNIOR",
+  2: "ADVANCED JUNIOR",
+  3: "INTERMEDIATE",
+  4: "SENIOR",
+  5: "ADVANCED SENIOR",
+}
+const details = {
+  1: "Do not have any experience on interviews",
+  2: "Had one or two interviews before",
+  3: "Had a few job interviews, but need more practice",
+  4: "Had seven to twelve interviews before, and want to know more about interview skills",
+  5: "Had more than twelve interviews before, and know much about interview skills",
+}
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      password: "",
+      openSnack: false,
+      severity: "error",
+      message: "Invalid email or password!",
+      language: "React",
+      experience: "",
+      value: 2,
+      hover: -1,
+    };
+  }
+
+  componentDidMount() {
+    fetch("/welcome")
+      .then((res) => {
+        console.log(res);
+        if (res.status === 200) return res.json();
+        else throw Error("Couldn't connect to the server");
+      })
+      .then((res) => {})
+      .catch((err) => {
+        console.log(err.message);
+      });
+  }
+
+  onChangeEmail = (e) => {
+    this.setState({ email: e.target.value });
+  };
+  onChangePassword = (e) => {
+    this.setState({ password: e.target.value });
+  };
+  signUp = () => {
+    this.props.history.push({
+      pathname: "/signup",
+      state: {},
+    });
+  };
+  handleClick = () => {
+//    this.validation();
+    this.setState({ openSnack: (this.state.experience === "" ) ? true : false });
+  };
+  async validation() {
+    //fetch here
+    const res = fetch("/signin", {
+      method: "post",
+      headers: { "Content-Type": "application/json" },
+      body: {
+        email: this.state.email,
+        password: this.state.password,
+      },
+    })
+      .then((response) => response.json())
+      .then((responseJson) => {
+        return responseJson.cookie;
+      })
+      .catch((error) => {
+        console.error(error);
+      });
+
+    var success = false;
+    if (true) {
+      success = true;
+    }
+    await this.setState({
+      severity: success ? "success" : "error",
+      message: success ? "Sign in successfully!" : "Invalid email or password!",
+    });
+    if (success === true) {
+    }
+  }
+  handleClose = (event, reason) => {
+    if (reason === "clickaway") {
+      return;
+    }
+    this.setState({ openSnack: false });
+  };
+  languageChange = (e) => {
+    this.setState({language: e.target.value});
+  };
+  experienceChange = (e) => {
+    this.setState({experience: e.target.value});
+  };
```

Comment:
Looks like a lot of these methods are taken from the Login component, when they shouldn't be here. Please remove them, it's also making it harder to read the code

Reply:
Removed.